src/features/game/expansion/components/Water.tsx

```diff
@@ -0,0 +1,79 @@
+import React from "react";
+
+import { GRID_WIDTH_PX } from "features/game/lib/constants";
+
+import { Section } from "lib/utils/hooks/useScrollIntoView";
+
+import dragonfly from "assets/decorations/dragonfly.gif";
+import Shark from "./water/Shark";
+
+import goblinSwimming from "assets/npcs/goblin_swimming.gif";
+import goblinSnorkling from "assets/npcs/goblin_snorkling.gif";
+import swimmer from "assets/npcs/swimmer.gif";
+
+export const Water: React.FC = () => {
+  return (
+    // Container
+    <div
+      style={{
+        height: "inherit",
+        width: "calc(100%)",
+        left: "0",
+        top: "0",
+      }}
+      className="absolute"
+    >
+      {/* Above Land */}
+      <Shark side="top" />
```

Comment:
The shark pixels appear much larger than the water pixels. For consistency we need to ensure that all of the pixels are the same size.

I would suggest scaling the shark down so it is the same size as some of the pixels you see on the land - such as the pebble or crop.

<img width="303" alt="Screen Shot 2022-07-08 at 9 02 47 am" src="https://user-images.githubusercontent.com/11745561/177885655-91e79967-cbf6-44c1-a9d4-774acbf3f45c.png">